### DISCUSSION TITLE
is it possible to add min-h-[calc(100vh - pt-16)], the calc value is className its self?

### DISCUSSION THREAD

**Discussion by crapthings** [tailwindlabs/tailwindcss#11324] [Ideas]
pt-16 is 4rem

min-h-[calc(100vh - 4rem)] but is it possible to use className?

**Comment by wongjn** [comment]
No. The closest you could do is use `theme()` to reference the value that `pt-16` uses, like `min-h-[calc(100vh-theme(spacing.16))]`.

**Reply by gndelia** [reply]
Does `theme()` work to get an extended value? For instance, I added a custom `h-footer` class, with the height defined in pixels in my tailwind config. Can I get `theme(height.footer)` or something like that?

**Comment by Carlgonzalez** [comment]
No, it is not possible to use a className as a value in a CSS `calc` function. The The calc function in CSS only accepts numerical values, and it does not evaluate class names or any other dynamic content. 
In your example, pt-16 is likely a class name representing a specific [MCDVOICE](https://www.mcdvoice.onl/) padding value in your CSS framework or custom styles. However, the calc function cannot directly interpret or substitute class names.
To achieve the desired effect, you would need to either manually calculate the value of pt-16 and use it in the calc function, or use JavaScript to dynamically calculate and apply the desired height value.
